provisioning/roles/rmt/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+- name: Install RMT via [composer]
+  shell: composer global require "liip/rmt"
+
+- name: Symlink RMT in /usr/local/bin
+  file: src=/home/vagrant/.composer/vendor/liip/rmt/RMT dest=/usr/local/bin/RMT state=link
```

Comment:
I guess that maybe there is a variable to reference the home directory. Isn't it?